docs/cugraph/source/nx_cugraph/nx_cugraph.md

```diff
@@ -44,25 +43,25 @@ user@machine:/# ipython bc_demo.ipy
 
 You will observe a run time of approximately 7 minutes...more or less depending on your cpu.
 
-Run the command again, this time specifiying cugraph as the NetworkX backend of choice.
+Run the command again, this time specifiying cugraph as the NetworkX backend.
 ```
 user@machine:/# NETWORKX_BACKEND_PRIORITY=cugraph ipython bc_demo.ipy
 ```
 This run will be much faster, typically around 20 seconds depending on your GPU.
 ```
 user@machine:/# NETWORKX_BACKEND_PRIORITY=cugraph ipython bc_demo.ipy
 ```
-There is also an option to add caching. This will dramatically help performance when running multiple algorithms on the same graph.
+There is also an option to cache the graph conversion to GPU. This can dramatically improve performance when running multiple algorithms on the same graph.
 ```
-NETWORKX_BACKEND_PRIORITY=cugraph CACHE_CONVERTED_GRAPH=True ipython bc_demo.ipy
+NETWORKX_BACKEND_PRIORITY=cugraph CACHE_CONVERTED_GRAPHS=True ipython bc_demo.ipy
```

Comment:
```suggestion
NETWORKX_BACKEND_PRIORITY=cugraph NETWORKX_CACHE_CONVERTED_GRAPHS=True ipython bc_demo.ipy
```

Reply:
Oh wow, that's a bad one.  Thanks for finding that.